packages/pangea-sdk/pyproject.toml

```diff
@@ -19,7 +19,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.7.2"
 requests = "^2.31.0"
-cryptography = "41.0.5"
+cryptography = "41.0.6"
```

Comment:
Do you think we can unpin it too or nah?

Reply:
Yes, unpinned.